packages/material-ui/src/Tooltip/Tooltip.test.js

```diff
@@ -30,14 +25,21 @@ function simulatePointerDevice() {
   document.dispatchEvent(new window.Event('pointerdown'));
 }
 
+// See handleClose in Tooltip
+const LEAVE_CONSTANT = 800;
```

Comment:
What's this used for? We now encode its "const"-nature three times: `const` keyword, uppercase snakecase, `_CONSTANT` suffix.

Reply:
There is a hardcoded 800 delay applied to the leaveDelay in Toootip. The name was trying to represent this fact.

Reply:
What about this strategy: We customize the default values in the `defaultProps` that we apply to the test cases. We use the same default props in the assertions (not the raw value).

This way, we can remove the usage of arbitrary values (`DEFAULT_ENTER_DELAY`, etc) and be resilient to future changes.

Reply:
Please no more `defaultProps` in tests. Makes them harder to read. I don't see any harm inlining these values in tests. Looks like they can easily be c&p.

Reply:
Inlining sounds better too 👍 (forcing a value for each test case)

Reply:
After the removal of setting enterDelay to 0. It makes sense not to use defaultProps anymore.